main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/component-to-devfile/oc_client"
+
+	"github.com/component-to-devfile/che_client"
+	"github.com/component-to-devfile/translate"
+	"github.com/gorilla/mux"
+	"gopkg.in/yaml.v2"
+)
+
+// HandlerCreateWorkspace create workspace using provided component CR
+func HandlerCreateWorkspace(w http.ResponseWriter, r *http.Request) {
+	v := mux.Vars(r)
+	ocClient := oc_client.OcClient{}
+	cr, err := ocClient.GetComponentDesc(v["component_name"])
+
+	t := translate.Translater{}
+	t.Input = cr
+	dF, err := t.Convert()
+	if err != nil {
+		json.NewEncoder(w).Encode(map[string]string{
+			"response": "devfile creation failed " + err.Error(),
+		})
+		return
```

Comment:
You should set a response code like `w.WriteHeader(http.StatusInternalServerError)` before returning.